cheroot/server.py

```diff
@@ -1353,7 +1354,27 @@ def close(self):
         self.rfile.close()
 
         if not self.linger:
-            self._close_kernel_socket()
+            # Make sure we terminate the connection at the transport level.
```

Comment:
This block reads easier when there's a method encapsulating the purpose. I recommend bringing back `_close_kernel_socket` but just change its docstring to `Terminate the connection at the transport level.`

Reply:
Fixed in 8664cd2e.